### PR TITLE
cloudprovider: add TencentCloud Provider

### PR DESCRIPTION
**Pull request by alphajc**
This PR add the [tencentcloud](https://cloud.tencent.com/) cloud provider for the cluster autoscaler.

### CODE REVIEW

**Comment by alphajc** [comment]
I signed it

File: cluster-autoscaler/go.mod [hunk 1]
```diff
@@ -22,6 +22,11 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
+	github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/as v1.0.335
```
Comment (mwielgus) [inline cluster-autoscaler/go.mod:25]: We don't accept any new dependencies for new cloud providers. Your cloud provider should have all the necessary stuff inside the cloud provider src directory. Please check other cloud providers for reference (example: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/alicloud)

**Review by mwielgus — CHANGES_REQUESTED** [review]

File: cluster-autoscaler/go.mod (hunk 1 above)
Comment (alphajc) [reply cluster-autoscaler/go.mod]: I have solved it.

**Review by mwielgus — CHANGES_REQUESTED** [review]
Please put yourself in the OWNERS file (possibly commented out if you are not in k8s org), so that we know who to contact for reviews, fixes, etc.

**Comment by alphajc** [comment]
> Please put yourself in the OWNERS file (possibly commented out if you are not in k8s org), so that we know who to contact for reviews, fixes, etc.

i have do it, but i want to know what does `[do-not-merge/invalid-owners-file](https://github.com/kubernetes/autoscaler/labels/do-not-merge%2Finvalid-owners-file)` mean ?

**Comment by alphajc** [comment]
> Please put yourself in the OWNERS file (possibly commented out if you are not in k8s org), so that we know who to contact for reviews, fixes, etc.

Please help to confirm if it is ok.

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Comment by alphajc** [comment]
@mwielgus I need a LGTM label, Please!

**Comment by alphajc** [comment]
\lgtm

**Review by feiskyer — APPROVED** [review]
/lgtm

**Pull request merged**